client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@material-ui/core';
+import AppBar from '@material-ui/core/AppBar';
+import Button from '@material-ui/core/Button';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useStoreState } from '../../store/hooks';
+import { removeItem, USER_SESSION_UUID_KEY } from '../../utils/storage';
+import { useWebSocket, WebSocketMessage } from '../../websocket';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    title: {
+      cursor: 'pointer',
+    },
+  })
+);
+
+const Navbar = () => {
+  const { push } = useHistory();
+  const classes = useStyles();
+  const webSocket = useWebSocket();
+  const username = useStoreState(state => state.user.username)
+
+  const onLogout = async () => {
+    removeItem(USER_SESSION_UUID_KEY);
+    webSocket.send(WebSocketMessage.logout())
+  }
+
+  return (
+    <>
+      <div className={classes.root}>
+        <AppBar position="static" color="transparent">
+          <Toolbar>
+            <Typography onClick={() => push('/')} variant="h6" className={classes.title}>
+              Fill in the blank
```

Comment:
Maybe even `Fill in the _____` replacing 'blank' with _blank_. The name isn't probably final. (Saying that now it might be final 😄)
The idea was to create at least provisional name since Cards Against Humanity is copyrighted.

Reply:
sure, why not...I mean, it's not important at the moment but we could create an issue for this _design_ part ;)